src/components/02_atoms/Widgets/BooleanCheckbox.js

```diff
@@ -5,7 +5,9 @@ import FormControlLabel from '@material-ui/core/FormControlLabel';
 import WidgetPropTypes from '../../05_pages/NodeForm/WidgetPropTypes';
 
 const BooleanCheckbox = props => {
-  const { onChange, label, id, value, ...checkboxProps } = props;
+  const { onChange, label, value } = props;
+  const checkboxProps = Object.assign({}, props);
+  delete checkboxProps.entityTypeId;
```

Comment:
I still think this should be done on a higher level - we never should have passed this prop in the first place. I think this will get out of control if we don't control the props set for widgets.

Reply:
Makes sense though, that's what I mentioned in the issue as well. If we are not going to be able to use, then why pass it.

Reply:
Exactly 👍

Reply:
Need to double check though if there is any impact, because if we might need to remove this prop from here: https://github.com/jsdrupal/drupal-admin-ui/blob/master/src/components/05_pages/NodeForm/index.js#L101

And this will have it removed from all of the widgets where FileUploadWidget seem to be using this as parameter: https://github.com/jsdrupal/drupal-admin-ui/blob/master/src/components/02_atoms/FileUpload/FileUpload.js#L56

Reply:
We could create object with key `metadata` or something similar that could contain this type of information. 🙂